src/library/crate/cratefeature.cpp

```diff
@@ -152,7 +152,10 @@ QString CrateFeature::formatRootViewHtml() const {
     html.append("</td><td rowspan=\"2\">");
     html.append("<img src=\"qrc:/images/library/crates_art.png\">");
     html.append("</td></tr>");
-    html.append(QString("<tr><td><a href=\"create\">%1</a>")
+    //Colorize links in lighter blue, instead of QT default dark blue.
+    //Links are still different from regular text, but readable on dark/light backgrounds.
+    //https://bugs.launchpad.net/mixxx/+bug/1744816
+    html.append(QString("<tr><td><a style=\"color:#0850D0;\" href=\"create\">%1</a>")
```

Comment:
I think this could be brighter still.

Reply:
``#0850D0`` 
looks pretty readable to me ( on OSX system-wide available as AlternateSelectedControlColor, used in Mixxx when selecting a library item.)
``#0496FF`` would be a similar color, if you think this still has to change.


![text_color](https://user-images.githubusercontent.com/4525897/36644303-6829ea0e-1a58-11e8-8b8f-c6370918bdc8.png)

Reply:
`#0496FF` is easy to read even with my brightness all the way down :+1: